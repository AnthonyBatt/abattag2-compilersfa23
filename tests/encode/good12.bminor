"\0x25\n0x25\n%\n"
