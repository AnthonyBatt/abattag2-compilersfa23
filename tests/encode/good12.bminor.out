
For the purpose of being able to see tabs, spaces, etc. printed I'm delimitng the start and end of the printed strings with ><

decoded string: >%
0x25
%
<
encoded string >"%\n0x25\n%\n"<
